Meta/gn/secondary/Ladybird/BUILD.gn

```diff
@@ -2,6 +2,10 @@ import("//Ladybird/compile_qt_resource_file.gni")
 import("//Ladybird/link_qt.gni")
 import("//Ladybird/moc_qt_objects.gni")
 
+declare_args() {
+  enable_qt = current_os != "mac"
```

Comment:
I think (almost?) all the other declare_args() have a comment above them to make the gn help args or w/e output look fancy.

Reply:
TIL GN does that 🙃 
```bash
# gn args --list=enable_qt Build
enable_qt
    Current value (from the default) = false
      From //Ladybird/BUILD.gn:7

    Build the Ladybird application using the Qt chrome.
```